source/aks/patching-flux-example.html.md.erb

```diff
@@ -93,6 +93,12 @@ flux install --components-extra image-reflector-controller,image-automation-cont
 
 You can review this [pull request](https://github.com/hmcts/sds-flux-config/pull/4270/files) and see the extra components are present.
 
+To ensure that the image automation components are working, as well as the [other checks](patching-aks-apps.html#checks-to-see-if-upgrade-worked-correctly) you can perform, you can check the flux repo to see if updates are being pushed by fluxcdbot.
```

Comment:
Maybe adding a page with general info on image automation would be better - just because there's nothing in any of those resources that would indicate whether an upgrade worked or not. I can sort either way.